docs/docs/genai/eval-monitor/legacy-llm-evaluation.mdx

```diff
@@ -6,967 +6,335 @@ import Tabs from "@theme/Tabs";
+    <td><APILink fn="mlflow.metrics.genai.answer_similarity">`answer_similarity`</APILink>, <APILink fn="mlflow.metrics.genai.answer_correctness">`answer_correctness`</APILink>, <APILink fn="mlflow.metrics.genai.answer_relevance">`answer_relevance`</APILink>, <APILink fn="mlflow.metrics.genai.relevance">`relevance`</APILink>, <APILink fn="mlflow.metrics.genai.faithfulness">`faithfulness`</APILink></td>
+    <td>Use the new [built-in LLM scorers](http://localhost:3000/genai/eval-monitor/scorers/llm-judge/predefined/#available-scorers). If the metric is not supported out of the box, define a custom LLM-as-a-Judge scorer using the <APILink fn="mlflow.genai.judges.make_judge">`make_judge`</APILink> API, following the example below.</td>
+  </tr>
+  <tr>
+    <td>Custom LLM-as-a-Judge metrics</td>
+    <td><APILink fn="mlflow.metrics.genai.make_genai_metric">`make_genai_metric`</APILink>, <APILink fn="mlflow.metrics.genai.make_genai_metric_from_prompt">`make_genai_metric_from_prompt`</APILink></td>
+    <td>Use the <APILink fn="mlflow.genai.judges.make_judge">`make_judge`</APILink> API to define a custom LLM-as-a-Judge scorer, following the example below.</td>
+  </tr>
 </table>
 
-#### Passing Inference Parameters
+#### Example of custom LLM-as-a-Judge metrics
 
-You can pass additional inference parameters such as `max_tokens`, `temperature`, `n`, to the model endpoint by setting the `inference_params` argument in <APILink fn="mlflow.evaluate" />. The `inference_params` argument is a dictionary that contains the parameters to be passed to the model endpoint. The specified parameters are used for all the input record in the evaluation dataset.
-
-:::note
-When your input is a dictionary format that represents request payload, it can also include the parameters like `max_tokens`. If there are overlapping parameters in both the `inference_params` and the input data, the values in the `inference_params` will take precedence.
-:::
-
-#### Examples
-
-**Chat Endpoint hosted by a local** [MLflow AI Gateway](/genai/governance/ai-gateway)
+The new evaluation API supports defining custom LLM-as-a-Judge metrics from a custom prompt template. This eliminates a lot of complexity and over-abstractions from the previous `make_genai_metric` and `make_genai_metric_from_prompt` APIs.
 
 ```python
-import mlflow
-from mlflow.deployments import set_deployments_target
-import pandas as pd
-
-# Point the client to the local MLflow AI Gateway
-set_deployments_target("http://localhost:5000")
-
-eval_data = pd.DataFrame(
-    {
-        # Input data must be a string column and named "inputs".
-        "inputs": [
-            "What is MLflow?",
-            "What is Spark?",
-        ],
-        # Additional ground truth data for evaluating the answer
-        "ground_truth": [
-            "MLflow is an open-source platform ....",
-            "Apache Spark is an open-source, ...",
-        ],
-    }
+from mlflow.genai.scorers import make_judge
```

Comment:
Seems make_judge doesn't exist in mlflow.genai.scorers namespace? While 'mlflow.genai.judges.make_judge' docstring include a deprecation warning to in favor of 'mlflow.genai.make_judge', should we fix the namespace?